scala-package/spark/src/test/scala/org/apache/mxnet/spark/SharedSparkContext.scala

```diff
@@ -80,14 +80,18 @@ trait SharedSparkContext extends FunSuite with BeforeAndAfterEach with BeforeAnd
     System.getProperty("user.dir")
   }
 
-  private def getJarFilePath(root: String): String = {
-    val jarFiles = new File(s"$root/target/").listFiles(new FileFilter {
+  private def findJars(root: String): Array[File] = {
+    val excludedSuffixes = List("bundle", "src", "javadoc", "sources")
+    new File(root).listFiles(new FileFilter {
       override def accept(pathname: File) = {
         pathname.getAbsolutePath.endsWith(".jar") &&
-          !pathname.getAbsolutePath.contains("bundle") &&
-          !pathname.getAbsolutePath.contains("src")
+          excludedSuffixes.map(!pathname.getAbsolutePath.contains(_)).forall(identity)
```

Comment:
You can just use excludedSuffixes.forall(!pathname.getAbsolutePath.contains(_))

Reply:
Done.